wpe/src/main/cpp/Browser/Browser.h

```diff
@@ -22,50 +22,41 @@
 
 #pragma once
 
-#include "ExportedBuffer.h"
 #include "MessagePump.h"
-#include "Page.h"
-#include "PageEventObserver.h"
-#include "PageSettings.h"
-
-#include <functional>
-#include <string>
-#include <unordered_map>
 
 #include <wpe/webkit.h>
-#include <wpe/wpe.h>
-
-struct ANativeWindow;
 
 class Browser final {
 public:
-    static Browser& instance()
+    static void configureJNIMappings();
+
+    static Browser& getInstance() noexcept
     {
-        static std::unique_ptr<Browser> singleton(new Browser());
-        return *singleton;
+        static Browser s_singleton;
+        return s_singleton;
     }
 
     Browser(Browser&&) = delete;
     Browser& operator=(Browser&&) = delete;
     Browser(const Browser&) = delete;
     Browser& operator=(const Browser&) = delete;
 
-    ~Browser() { shut(); }
+    ~Browser() { jniShut(); }
 
-    WebKitWebContext* webContext() const { return m_webContext; }
+    WebKitWebContext* getWebContext() const noexcept { return m_webContext.get(); }
```

Comment:
this is not according to webkit code style

https://webkit.org/code-style-guidelines/#names-setter-getter

Reply:
I was trying to be coherent with the Android/Java part, but you're right.
I'm fixing it.